extensions/amp-next-page/validator-amp-next-page.protoascii

```diff
@@ -85,3 +85,28 @@ tags: {  # <amp-next-page src>
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-next-page"
 }
+tags: {  # <amp-next-page type="adsense">
+  html_format: AMP
+  tag_name: "AMP-NEXT-PAGE"
+  spec_name: "amp-next-page with AdSense content recommendations"
```

Comment:
I'd recommend renaming this to:

`spec_name: "amp-next-page [type=adsense]"`

We typically don't use sentences in the spec names.

Also, please alphabetize the attributes by name (e.g. type should come after slot).